zipkin-storage/cassandra-3/README.md

```diff
@@ -0,0 +1,61 @@
+# storage-cassandra-3
```

Comment:
I'm assuming you don't care about this.. I'm going to rename this to "cassandra3" as opposed to "cassandra-3" since it makes the jar look less weird and keeps the field number consistent vs other things